.mailmap

```diff
@@ -233,20 +265,21 @@ Matteo Andreozzi <matteo.andreozzi@arm.com> Matteo Andreozzi <Matteo.Andreozzi@a
 Matteo M. Fusi <matteo.fusi@bsc.es>
 Matt Evans <matt.evans@arm.com> Matt Evans <Matt.Evans@arm.com>
 Matthew Poremba <matthew.poremba@amd.com> Matthew Poremba <Matthew.Poremba@amd.com>
+Matthias Boettcher <matthias.boettcher@arm.com>
 Matthias Hille <matthiashille8@gmail.com>
-Matthias Jung <jungma@eit.uni-kl.de>
-Matthias Jung <matthias.jung@iese.fraunhofer.de>
+Matthias Jung <matthias.jung@iese.fraunhofer.de> Matthias Jung <jungma@eit.uni-kl.de>
 Matt Horsnell <matt.horsnell@arm.com> Matt Horsnell <matt.horsnell@ARM.com>
 Matt Horsnell <matt.horsnell@arm.com> Matt Horsnell <Matt.Horsnell@arm.com>
 Matt Horsnell <matt.horsnell@arm.com>Matt Horsnell <Matt.Horsnell@ARM.com>
 Matt Poremba <matthew.poremba@amd.com> Matt Poremba <Matthew.Poremba@amd.com>
-Matt Sinclair <mattdsinclair@gmail.com> Matthew Sinclair <matthew.sinclair@amd.com>
+Matt Sinclair <mattdsinclair.wisc@gmail.com> Matt Sinclair <mattdsinclair@gmail.com>
+Matt Sinclair <mattdsinclair.wisc@gmail.com> Matthew Sinclair <matthew.sinclair@amd.com>
```

Comment:
I think 276 and 277 are the same except for capitalization?

Reply:
Fixed it. Thanks.

Reply:
I am not sure I fully understand the need to have two very similar versions of my info here, but if this satisfies the Davis folks, fine with me.